src/test/kotlin/com/boclips/apidocs/HestuDocTests.kt

```diff
@@ -93,4 +97,92 @@ class HestuDocTests : AbstractDocTests() {
             .then()
             .assertThat().statusCode(CoreMatchers.`is`(200))
     }
+
+    @Test
+    @Disabled
+    fun `Get highlight`(restDocumentation: RestDocumentationContextProvider) {
+        val responseFields = responseFields(
+            fieldWithPath("getHighlight").description("Get a given highlight by ID"),
+        )
+
+        val pathParameters = RequestDocumentation.pathParameters(
+            RequestDocumentation.parameterWithName("highlightId").description("ID of the highlight"),
+        )
+
+        RestAssured.given(hestuDocumentationSpec)
+            .filter(
+                document(
+                    "highlight-get",
+                    pathParameters,
+                    responseFields,
+                ),
+            )
+            .filter(
+                RestAssuredRestDocumentationWrapper.document(
+                    "{method-name}",
+                    "Can get a given highlight by ID",
+                    false,
+                    responseFields,
+                ),
+            )
+            .`when`()
+            .get(HIGHLIGHT_PATH, productionVideoId).apply { prettyPrint() }
+            .then()
+            .assertThat().statusCode(CoreMatchers.`is`(200))
+    }
+
+    @Test
+    @Disabled
+    fun `Get highlights`(restDocumentation: RestDocumentationContextProvider) {
+        val responseFields = responseFields(
+            fieldWithPath("getHighlights").description("Get filtered highlights"),
+        )
+
+        // TODO("Review the types, descriptions, and optionality of these params w/ DS)
+        val requestParameters = RequestDocumentation.requestParameters(
+            RequestDocumentation.parameterWithName("threshold")
+                .optional()
+                .description("")
```

Comment:
```suggestion
                .description("Minimum similarity score a clip needs to be returned between 0 and 1.")
```
default is None (optional) which will return the closest matches no matter what.